CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ else()
     LANGUAGES C CXX)
 endif()
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")

```

Comment:
Is this really necessary?

Reply:
Not sure I will have to retest that one a bit. Or if someone with Windows could do it that would be a big help. My VM has terrible performance and a 30-day time limit.

Reply:
We've had LTO support auto-detected for a while now (since 2019: https://github.com/diasurgical/devilutionX/commit/332dbf71c1a3693244484518c87afb5a530be0dd) so I'd be surprised if is now necessary

Reply:
It doesn't detect when the compiler is buggy :P

Reply:
I'd thought we worked around that in the toolchain files but looks like that was only for MinGW